services/galley/test/integration/TestHelpers.hs

```diff
@@ -13,10 +13,7 @@ import qualified API.SQS             as SQS
 test :: IO TestSetup -> TestName -> TestM a -> TestTree
 test s n h = testCase n runTest
   where
-    assertClean :: TestM ()
-    assertClean = SQS.assertQueueEmpty
-
     runTest :: Assertion
     runTest = do
         setup <- s
-        void . flip runReaderT setup . runTestM $ h >> assertClean
```

Comment:
Not really sure what this was supposed to gain us? It works to just wipe the queues after every test; but maybe this is slow?